WTWSMS/decisions/realm_decisions.txt

```diff
@@ -2028,26 +2061,6 @@ decisions = {
 					value = off
 				}
 			}
-			
-			OR = {
-				ai = no
-				AND = {
-					trait = proud
-					prestige = 5000
-				}
-				AND = {
-					trait = ambitious
-					prestige = 5000
-				}
-				AND = {
-					realm_size = 70
-					any_demesne_title = {
-						count = 4
-						tier = DUKE
-					}
-				}
-				prestige = 15000
-			}
```

Comment:
**_NO. ABSOLUTELY NOT._**

This is the CUSTOM REALM CREATION, which is _**CRITICAL**_ to AI management. Right now, because of YOUR insistence on difficult, restrictive requirements for de jure titles, the ONLY way the AI can get a higher title in some locations is if he uses THIS Decision.

However, you accidently made THIS Decision impossible to take.

If this isn't changed- AND CHANGE IT MUST -the AI will never create Kingdoms or Empires in some swaths of the Map. That breaks immersion, balance, realism, history, and the point of the Decision.

I will absolutely tolerate nothing less than this one Decision being fixed. **_This is game-breaking for the AI in some regions._**